core/src/main/java/edu/wpi/grip/core/events/SourceStartedEvent.java

```diff
@@ -0,0 +1,19 @@
+package edu.wpi.grip.core.events;
+
+
+import edu.wpi.grip.core.Source;
+
+/**
+ * An event that occurs when a source is started.
```

Comment:
Also, I'm not sure what "started" means in this context.  It seems like in some cases (image source) this is sent every time the socket changes, but in some (webcam source) it's sent after the capture thread is started.